includes/html/pages/apps.inc.php

```diff
@@ -643,8 +643,9 @@
 ];
 
 $sep = '';
-$apps = LibreNMS\Util\ObjectCache::applications()->flatten();
+$apps = LibreNMS\Util\ObjectCache::applications();
 foreach ($apps as $app) {
+    $app = $app[0];
```

Comment:
This is not an array, probably should use first.
```suggestion
foreach ($apps as $app_group) {
    $app = $app_group->first();
```

Reply:
this still seems like a bit of a weird way to fix the issue.